content/en/docs/concepts/windows/intro.md

```diff
@@ -352,6 +352,37 @@ Windows Server SAC release
 
 The Kubernetes [version-skew policy](/docs/setup/release/version-skew-policy/) also applies.
 
+## Worker Node Hardware Recommendations {#windows-node-hardware-recommendations}
+
+The minimum hardware requirements for a Windows worker node depend on several
+factors, including the Windows operating system version and specific workload
+requirements. Mentioned below are the minimum system requirements, keeping in
+mind that they can vary based on the components installed on the node and the
+workloads required to run on it.
+
+- 1.4 GHz 64-bit processor, supporting virtualization
+- 8GB or more of RAM
```

Comment:
The https://learn.microsoft.com/en-us/azure/aks/concepts-clusters-workloads  discusses interesting calculations per nodes on AKS w.r.t. memory and cpu reservation, so it may be worth to point at it from this guidelines.

Reply:
@mloskot, I appreciate your input. The calculations on the AKS webpage are potentially useful but I have reservations about including them in the documentation. My concern is that some of this information is specific to AKS e.g. the AKS kube-reserved values and how they are considered in determining capacity. Also, I would suggest to be slightly vague with the intention of keeping things simple and not provide advanced calculations. What are your thoughts on this?

Reply:
@tzifudzi Good point, but I meant to use those guidelines as additional input to work out K8s own guidelines for Windows, not to include those verbatim.

Reply:
Oh ok got it! I assumed you meant the latter. I for sure will take a look for inspiration on relevant docs to include. Thanks for the suggestion